lib/internal/module.js

```diff
@@ -83,43 +83,50 @@ const builtinLibs = [
   'string_decoder', 'tls', 'tty', 'url', 'util', 'v8', 'vm', 'zlib'
 ];
 
+// Make built-in modules available directly (loaded lazily).
 function addBuiltinLibsToObject(object) {
-  // Make built-in modules available directly (loaded lazily).
-  builtinLibs.forEach((name) => {
+  const configurable = true;
+  const enumerable = false;
+
+  // Setter function that will be bound to some lib name
+  function setReal(libname, value) {
+    // Deleting the property before re-assigning it disables the
+    // getter/setter mechanism.
+    delete object[libname];
+    object[libname] = value;
+  }
+
+  for (var n = 0, len = builtinLibs.length; n < len; n++) {
```

Comment:
No need to cache the length, V8 can work fine with using `.length` in the conditional part.